server/routes/oembed/chart.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Endpoint for iframe-friendly embed-able chart pages"""
+
+from flask import Blueprint
+from flask import render_template
+from flask import request
+
+from server import cache
+import server.lib.config as libconfig
+
+DEFAULT_WIDTH = 500
+DEFAULT_HEIGHT = 400
+ALLOWED_CHART_TYPES = [
+    "bar",
+    "gauge",
+    "line",
+    "map",
+    "pie",
+    "ranking",
+]
+
+# Define blueprint
+bp = Blueprint("chart", __name__, url_prefix="/chart")
+
+hostname = "https://datacommons.org"
+cfg = libconfig.get_config()
```

Comment:
nit: usually when trying to read config values, can use current_app.config like https://github.com/datacommonsorg/website/blob/master/server/routes/disaster/html.py#L46 where you just read the value instead of running a whole function to get the config

Reply:
Thanks for the pointer! Added.